opa/Dockerfile

```diff
@@ -58,13 +58,32 @@ WORKDIR /daemontools/admin/daemontools-${DAEMONTOOLS_VERSION}
 
 RUN package/install
 
-FROM stackable/image/vector
+FROM stackable/image/vector as opa-builder
 
 ARG PRODUCT
 ARG RELEASE
 ARG TARGETARCH
 ARG TARGETOS
 
+ENV GOARCH=$TARGETARCH
+ENV GOOS=$TARGETOS
+
+RUN microdnf update && \
+    microdnf install git go gzip tar
```

Comment:
```suggestion
    microdnf install --assumeyes \
        git \
        go \
        gzip \
        tar && \
        microdnf clean all
```

Reply:
We use as convention one line per installed tool. I would personally appreciate a comment why / for what you install those tools.

Reply:
Yes, Maxi is right.
Please split it across multiple lines.
`--assumeyes` is not needed. That is now baked into the base image and also yes: Please add a comment for why everything is needed. Thanks!

Reply:
Will do. Is `microdnf clean all` really needed? This is just a builder image, so it gets thrown away anyways